### PR TITLE
V 2.2.0

### PR DESCRIPTION
**Pull request by EricDupertuis**
Migrate to VanillaJs, no need for jQuery as a dependencies

### CODE REVIEW

**Comment by EricDupertuis** [comment]
Changements effectués, par contre pas besoin de checker style.display !== none en plus du offsetWidth & offsetHeight.
Ca implique display: none si offsetWidth et offsetHeight sont les deux plus > 0
screenshot from 2017-10-24 14-07-04

File: Resources/public/js/dfp.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ function dfp(slots, pageTarget, debug) {
     var dfpSlots = {};
     // Declare each slot
     document.querySelectorAll('[data-dfp]').forEach(function (e) {
-        var isSplash = e.setAttribute('data-splashcreen', true);
+        var isSplash = e.getAttribute('data-splashcreen', true);
```
Comment (ragusa87) [inline Resources/public/js/dfp.js]: var isSplash = e.getAttribute('data-splashcreen') === "true";

File: Resources/public/js/dfp.js [hunk 2]
```diff
@@ -1,5 +1,26 @@
+function setCookie(name,value,days) {
```
Comment (ragusa87) [inline Resources/public/js/dfp.js]: function setCookie(name,value,expiration) {
 +    var expires = "";
 +    if (expiration) {
 +        expires = "; expires=" + expiration.toUTCString();
 +    }
 +    document.cookie = name + "=" + value + expires + "; path=/";
 +}

File: Resources/public/js/dfp.js [hunk 3]
```diff
@@ -135,7 +156,7 @@ function dfp(slots, pageTarget, debug) {
             // create cookie
             var expireDate = new Date();
             expireDate.setMilliseconds(expireDate.getMilliseconds() + 5 * 60000); // 5min
-            Cookies.set("no-splashscreens", "1", {expires: expireDate});
+            setCookie("no-splashscreens", "1", {expires: expireDate});
```
Comment (ragusa87) [inline Resources/public/js/dfp.js]: +            setCookie("no-splashscreens", "1", expireDate);

**Pull request merged**